cos_registration_server/applications/tests.py

```diff
@@ -26,17 +26,30 @@
     "playbackConfig": {"speed": 1},
 }
 
-SIMPLE_ALERT_RULE = """
+SIMPLE_PROMETHEUS_ALERT_RULE = """
     groups:
         - name: cos-robotics-model_robot_test_%%juju_device_uuid%%
         rules:
             - alert: MyRobotTest_{{ $cos.instance }}
-            annotations:
-            description: "The very custom description"
-            summary: Not enough memory alert (instance {{ $labels.instance }})
-            expr: (node_memory_MemFree_bytes{device_instance="${{ $cos.instance }}"})/1e9 < 30
-            for: 5m
-            severity: critical
+              annotations:
+              description: "The very custom description"
+              summary: Not enough memory alert (instance {{ $labels.instance }})
+              expr: (node_memory_MemFree_bytes{device_instance="${{ $cos.instance }}"})/1e9 < 30
+              for: 5m
+              severity: critical
+"""
+
+SIMPLE_LOKI_ALERT_RULE = """
+    groups:
+        - name: cos-robotics-model_high_log_rate_per_instance
+	    rules:
```

Comment:
I think the `rules:` tag must be at the same level as the `-name: `.
It should be:
```
groups:
  - name: lkajf
    rules:
```